doc/deoplete.txt

```diff
@@ -267,16 +267,20 @@ KEY MAPPINGS 					*deoplete-key-mappings*
 
 					*deoplete#mappings#manual_complete()*
 deoplete#mappings#manual_complete([{sources}])
-		Use this function on inoremap <expr>.  The keymapping call the
-		completion of deoplete.  When you rearrange the completion
-		of the Vim standard, you use it.
-		If you give {sources} argument, deoplete will call {sources}.
-		{sources} is name of source or list of sources name.
+		Use this function with |:inoremap| <expr> (|:map-expression|).
+		It calls the completion of deoplete.  You can use it with
+		custom completion setups.
+		You can provide a list of {sources}: It can be the name of a
+		source or a list of sources name.
 >
-		inoremap <silent><expr><Tab>
+		See also |g:deoplete#disable_auto_complete|, which should be
```

Comment:
You mean the reference to `g:deoplete#disable_auto_complete`?

It should be used if you want to trigger the completion manually, no?